SPECS/bcc.spec

```diff
@@ -10,42 +10,37 @@ License:        ASL 2.0
 URL:            https://github.com/iovisor/bcc
 Source0:        bcc.tar.gz
 
-BuildArch:      x86_64
-BuildRequires:  bison, cmake >= 2.8.7, flex, gcc, gcc-c++, python2-devel, elfutils-libelf-devel-static
+ExclusiveArch: x86_64
+BuildRequires: bison cmake >= 2.8.7 flex
+BuildRequires: gcc gcc-c++ python2-devel elfutils-libelf-devel-static
+BuildRequires: luajit luajit-devel
+BuildRequires: llvm-devel llvm-static
+BuildRequires: clang-devel
```

Comment:
Once doing this, don't you also need a `Requires: clang-libs` under libbcc to make up for the new .so requirement?